File: Cliente/Cliente/Transformadores/FechaABlanco.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Windows.Data;

namespace Trascend.Bolet.Cliente.Transformadores
{
    class FechaABlanco : IValueConverter
    {
        public object Convert(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture)
        {
            string fecha = value.ToString();

            return fecha.Equals("01/01/0001 12:00:00 a.m.") ? "" : ((DateTime)value).ToString("dd/MM/yyyy"); 
        }

        public object ConvertBack(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture)
        {
            throw new NotImplementedException();
        }
    }
}

File: trunk/src/Cliente/Cliente_Fac/Transformadores/FechaABlanco.vb
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Windows.Data
Imports System.Windows.Controls
Namespace Transformadores
    Public Class FechaABlanco
        Implements IValueConverter

        Public Function Convert(ByVal value As Object, ByVal targetType As System.Type, ByVal parameter As Object, ByVal culture As System.Globalization.CultureInfo) As Object Implements System.Windows.Data.IValueConverter.Convert
            Dim fecha As String = value.ToString()

            Return If(fecha.Equals("01/01/0001 12:00:00 a.m."), "", CType(value, DateTime).ToString("dd/MM/yyyy"))
        End Function

        Public Function ConvertBack(ByVal value As Object, ByVal targetType As System.Type, ByVal parameter As Object, ByVal culture As System.Globalization.CultureInfo) As Object Implements System.Windows.Data.IValueConverter.ConvertBack
            Throw New NotImplementedException()
        End Function
    End Class
End Namespace
